packages/rocketchat-lib/server/methods/startEmailIntercepter.js

```diff
@@ -0,0 +1,28 @@
+Meteor.methods({
+	startEmailIntercepter() {
+		if (RocketChat.settings.get('Direct_Reply_Enable') && RocketChat.settings.get('Direct_Reply_Protocol') && RocketChat.settings.get('Direct_Reply_Host') && RocketChat.settings.get('Direct_Reply_Port') && RocketChat.settings.get('Direct_Reply_Username') && RocketChat.settings.get('Direct_Reply_Password')) {
+			if (RocketChat.settings.get('Direct_Reply_Protocol') === 'IMAP') {
+				if (!RocketChat.IMAP || !RocketChat.IMAP.isActive()) {
+					RocketChat.IMAP = new RocketChat.IMAPIntercepter({
+						user: RocketChat.settings.get('IMAP_Username'),
```

Comment:
It won't start until older is stopped. `RocketChat.IMAP.isActive()` is `true` if there is any instance already running.